roles/dtc/common/templates/ndfc_vrf_lite.j2

```diff
@@ -1,6 +1,13 @@
 {% if item.ospf is defined and item.ospf %}
 {% include 'ndfc_vrf_lite_ospf.j2' %}
 {% endif %}
+{% set bgp = {} %}
 {% if item.bgp is defined and item.bgp %}
-{% include 'ndfc_vrf_lite_ebgp.j2' %}
-{% endif %}
\ No newline at end of file
+  {% set _ = bgp.update({'enabled': true}) %}
+{% endif %}
+{% if switch_item.bgp_peers or switch_item.bgp %}
+    {% set _ = bgp.update({'enabled': true}) %}
+{% endif %}
```

Comment:
Nit: Can we consolidate both of these if conditions into one, as both are setting the same variable/value? For example:
```
{% if (item.bgp is defined and item.bgp) or switch_item.bgp_peers or switch_item.bgp %}
  {% set _ = bgp.update({'enabled': true}) %}
{% endif %}
```

Reply:
i agree, thanks @devegupt !